src/Microsoft.Extensions.Hosting/Internal/HostProcessLifetime.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.Extensions.Hosting.Internal
+{
+    public class HostProcessLifetime : IHostLifetime
```

Comment:
If the other lifetime is called ConsoleLifetime, let's call this HostLifetime.

Reply:
ProcessLifetime would be more accurate.

Reply:
Oops typo, I meant that. ~~Host~~ProcessLifetime